bluepyopt/ephys/morphologies.py

```diff
@@ -94,7 +94,7 @@ def instantiate(self, sim=None, icell=None):
         # TODO this is to get rid of stdout print of neuron
         # probably should be more intelligent here, and filter out the
         # lines we don't want
-        sim.neuron.h.hoc_stdout('/dev/null')
+        sim.neuron.h.hoc_stdout('null')
```

Comment:
If I look at the Neuron documentation, it seems this will create a file called 'null' ?
Maybe we should put a switch statement here, to make sure that /dev/null is still used on all systems apart from Windows.